package.json

```diff
@@ -10,15 +10,17 @@
     "lodash": "^4.6.1"
   },
   "devDependencies": {
-    "async": "^2.0.0-rc.1",
+    "async": "^2.0.0",
     "chai": "^3.5.0",
     "mocha": "^2.4.5",
-    "mockgoose": "^6.0.0",
-    "mongodb": "^2.1.11",
-    "mongoose": "^4.4.9"
+    "mockgoose": "^7.0.0",
+    "mongoose": "^4.8.6"
```

Comment:
Yes. Peer dependency tells for the library consumer that he should provide such dependency. Dev dependencies are not installed when you npm install a package. The reason why there may not have been problem often in this case is probably because the library consumer already has added mongoose dependency. Nevertheless, mongoose is also a production dependency, and in order to avoid weird version mismatch problems between swagger-mongoose and the consuming project, it is best to have it as a peer dependency so that the library consumer can decide the actual version.

http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies